scipy/stats/_stats_py.py

```diff
@@ -8177,14 +8243,30 @@ def kstest(rvs, cdf, args=(), N=20, alternative='two-sided', method='auto'):
 
     Returns
     -------
-    statistic : float
-        KS test statistic, either D, D+ or D-.
-    pvalue : float
-        One-tailed or two-tailed p-value.
+    res: KstestResult
+        An object containing attributes:
+
+        statistic : float
+            KS test statistic, either D+, D-, or D (the maximum of the two)
+        pvalue : float
+            One-tailed or two-tailed p-value.
+        statistic_location: float
+            Value of `x` corresponding with the KS statistic; i.e. the
+            observation at which the difference between CDFs is measured.
+        statistic_sign: int
+            1 if the KS statistic is the maximal positive difference between
+            CDFs (D+), -1 if the KS statistic is the negative difference (D-).
+
+        statistic_sign: int
+            1 if the KS statistic is the maximal positive difference between
+            empirical and hypothesized distributions (D+), -1 if the KS
+            statistic is the negative difference (D-). If a 2-sample test is
+            run, the difference is between the first and second empirical
+            distributions.
```

Comment:
Please pick one.
If the latter, consider that the second sentence begins with "if a 2-sample test is run", but there is no corresponding part of the first sentence.